src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import { MerriageInfo } from '../MerriageInfo/MerriageInfo';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    {person.isMarried ? (
+      <MerriageInfo
+        message={`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
would be better to move this to the variable `person.sex === 'm' ? 'wife' : 'husband'`